src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
I agree that sourcing the setup files is the better approach, but given that Debian policy 9.9 I don't think that's acceptable.

I don't think it's a problem for older distros. Take Hydro, for example, Hydro users will get the updated `rospkg` that inserts `/usr/share` implicitly, but not the binary packages that set `RPP_NO_DEFAULTS`. But that shouldn't be a problem since there are no packages in `/usr` to mess with Hydro users. I guess there's a chance for breaking things in a way that I can't think of right now and for slower performance when searching for packages, but that's all. There's no obvious issue that I can see.

For packages with other env variables, they will need to be patched as they are put into Debian one-by-one. It doesn't affect our packages, but it does mean more work for the Debian packagers.